### DISCUSSION TITLE
Possibility to mutate the countries list

### DISCUSSION THREAD

**Discussion by Paulsky** [laravel/nova-issues#6366] [Ideas & Feature Requests]
Hello,

I would like to mutate the countries list in the Country field. Right now, the countries are hardcoded as options. I understand you use the ISO 3166 list. But this list is not always functional. Example: I need the country to be _Bonaire_ or _Saba_. But I can only select BQ (_'Bonaire, Sint Eustatius and Saba'_). So, I would like to remove 'BQ' and add `'BQ-BQ' => 'Bonaire'` and` 'BQ-SA' => 'Saba'`. Please see https://www.iso.org/obp/ui/#iso:code:3166:BQ

I have created a workaround in a `Trait,` but I'm wondering if there is another way?

```
public function countryField()
    {
        $options = [
            'AF' => 'Afghanistan',
            'AX' => 'Aland Islands',
            'AL' => 'Albania',
            'DZ' => 'Algeria',
            //.... COPIED OVER ALL $options on line 26 in Laravel\Nova\Fields\Country
        ];
        unset($options['BQ']);
        $options['BQ-BO'] = 'Bonaire';
        $options['BQ-SA'] = 'Saba';


        asort($options);

        return Country::make(__('location.country'), 'country')->options($options);
}
```

**Comment by crynobone** [comment]
You can create your own custom `Country` field.

**Comment by Paulsky** [comment]
Yes, I can
